auth/api/grpc/requests.go

```diff
@@ -5,7 +5,7 @@ package grpc
 
 import (
 	"github.com/mainflux/mainflux/auth"
-	"github.com/mainflux/mainflux/internal/apiutil"
+	initutil "github.com/mainflux/mainflux/internal/init"
```

Comment:
Why not just `"github.com/mainflux/mainflux/internal`?

Reply:
the package has been changed as per you proposal in https://github.com/mainflux/mainflux/pull/1601#discussion_r880587891